interface/orders/types_edit.php

```diff
@@ -366,6 +367,17 @@ class='inputtext' />
     style='width:100%' readonly />
   </td>
  </tr>
+ <tr>
+     <td nowrap> &nbsp;</td>
+  <td>
+      <label> 
+        <?php echo xlt('Activity'); ?>
+          <input type='checkbox'  name='form_activity' value='1' <?php if ($row['activity']) {
+              echo 'checked=""' ;
```

Comment:
For a new procedure type you want this to default to checked. Doesn't look like this will do that.